wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
+
+**Status:** Accepted|Draft|Other
+
+## Recommendation 
+
+- Formalize recommendation in title 
+- Provide a brief, 1-3 sentence summary of reasoning behind this change
+- Provide alternate recommendation(s) to the one in the title if needed 
+
+## Context/Precedents 
+
+If existing, provide any research, links to PR(s) from other communities, etc. that provide precedent for this decision. 
```

Comment:
I think it would help if the template references the evaluation framework. For example here, I'm interested in if this is a first-order or second-order concern and why.

Reply:
+1 referencing our framework

using master as an example, it clearly has first-order concerns, but third-order concerns are why we can't just search-replace it away

would it be relevant to link to PR's that have already merged within this community as precedent or prior art?